groups/cdn-terraform/variables.tf

```diff
@@ -0,0 +1,48 @@
+variable "aws_account" {
+  type        = string
+  description = "The name of the AWS account"
+}
+
+variable "aws_region" {
+  type        = string
+  description = "The AWS region in which resources will be created"
+}
+
+variable "bucket_name" {
+  type        = string
+  description = "Name of the S3 bucket"
+}
+
+variable "cloudfront_distribution_arn" {
+  type        = string
+  description = "ARN of the CloudFront distribution"
+}
```

Comment:
Can this variable be removed? It isn't referenced anywhere and the CloudFront distribution ARN can be obtained from the `aws_cloudfront_distribution` resource and is only known after resource creation.

```suggestion
```